sys/linux/sys.txt

```diff
@@ -914,6 +915,7 @@ open_flags = O_WRONLY, O_RDWR, O_APPEND, FASYNC, O_CLOEXEC, O_CREAT, O_DIRECT, O
 open_mode = S_IRUSR, S_IWUSR, S_IXUSR, S_IRGRP, S_IWGRP, S_IXGRP, S_IROTH, S_IWOTH, S_IXOTH
 resolve_flags = RESOLVE_BENEATH, RESOLVE_IN_ROOT, RESOLVE_NO_MAGICLINKS, RESOLVE_NO_SYMLINKS, RESOLVE_NO_XDEV
 madvise_flags = MADV_NORMAL, MADV_RANDOM, MADV_SEQUENTIAL, MADV_WILLNEED, MADV_DONTNEED, MADV_REMOVE, MADV_DONTFORK, MADV_DOFORK, MADV_HWPOISON, MADV_SOFT_OFFLINE, MADV_MERGEABLE, MADV_UNMERGEABLE, MADV_HUGEPAGE, MADV_NOHUGEPAGE, MADV_DONTDUMP, MADV_DODUMP, MADV_WIPEONFORK, MADV_KEEPONFORK, MADV_COLD, MADV_PAGEOUT
+process_madvise_flags = MADV_COLD, MADV_PAGEOUT, MADV_MERGEABLE, MADV_UNMERGEABLE
```

Comment:
The actual code in linux-next only supports MADV_COLD and MADV_PAGEOUT.

Reply:
Thx! I fixed all of the things mentioned (the MADV_MERGEABLE, MADV_UNMERGEABLE flags are supported on my local version of linux-next but looks like it got removed in a later version).